ckcp/openshift_dev_setup.sh

```diff
@@ -275,9 +275,19 @@ patches:
         value: $ckcp_route " >>"$ckcp_temp_dir/kustomization.yaml"
 
   echo -n "  - kcp $kcp_version: "
-  kubectl apply -k "$ckcp_temp_dir" >/dev/null 2>&1
+  kubectl apply -k "$ckcp_temp_dir" >/dev/null
   # Check if ckcp pod status is Ready
-  kubectl wait --for=condition=Ready -n $ns pod -l=app=kcp-in-a-pod --timeout=90s >/dev/null
+  kubectl wait --for=condition=Ready -n $ns pod -l=app=kcp-in-a-pod --timeout=90s >/dev/null 2>&1 || error="true"
+  if [ "$error" == "true" ]; then
+    printf "Debug: print out pod state\n"
+    kubectl -n $ns describe pods
```

Comment:
Hey @xinredhat, is this intentional to try to describe all the pods in the `$ns` NS?

Reply:
yes

Reply:
It is better if we still mention the name of the pod instead of describing all pods in that namespace (at the moment, there's only one pod but may change in the future).
we could use a local variable to get the name of the pod and then substitute it on line 281, 284 and 287